recipes/squadgen/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "squadgen" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ClimateGlobalChange/squadgen/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a39f9f467563e796c1b0a53c7bf7d046f50110a030d92350754a60ada4eda053
+
+build:
+  skip: true  # [not linux]
+  number: 0
+  ignore_run_exports:
+    # I don't really understand why these dynamic libraries aren't being linked in...
+    - libpng
+    - zlib
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - make
+  host:
+    - libnetcdf
+    - libnetcdf * nompi_*
+    - libpng
+    - zlib
+
+test:
+  files:
+    - test/grid.png
+  commands:
+    - >
+      SQuadGen --refine_file test/grid.png --refine_level 5
+      --refine_type LOWCONN --resolution 30 --smooth_type SPRING
+      --smooth_dist 3 --smooth_iter 20 --output grid.g
+    - test -f "grid.g"
+
+about:
+  home: https://github.com/ClimateGlobalChange/squadgen
+  license: GPL-2.0-or-later and CUSTOM
```

```suggestion
  license: GPL-2.0-or-later and NetCDF
```
Isn't this the right one?

---

Thanks you @xhochy!  These are very helpful suggestions!